sdk/textanalytics/Azure.AI.TextAnalytics/src/JobManifestTasks.cs

```diff
@@ -37,5 +37,10 @@ internal partial class JobManifestTasks
         /// SentimentAnalysisTasks
         /// </summary>
         internal IList<SentimentAnalysisTask> SentimentAnalysisTasks { get; set; }
+
+        /// <summary>
+        /// ExtractiveSummarizationTasks
+        /// </summary>
+        public IList<ExtractiveSummarizationTask> ExtractiveSummarizationTasks { get; set; }
```

Comment:
not that it really matters, but to go with the flow, u can make this type internal

Reply:
You mean the `ExtractiveSummarizationTask` type? It is generated as an internal type by default.